mono_repo/lib/src/github_config.dart

```diff
@@ -19,7 +22,59 @@ class GitHubConfig {
     Map<String, dynamic> on,
     String cron,
     this.workflows,
-  ) : on = _parseOn(on, cron);
+  ) : on = _parseOn(on, cron) {
+    if (workflows != null) {
```

Comment:
[optional] Consider flipping this with an early return to reduce indentation.

Reply:
ack